### PR TITLE
[draft] run ovs kernel mechanism test

### PR DESCRIPTION
**Pull request by pperiyasamy**
Signed-off-by: Periyasamy Palanisamy <periyasamy.palanisamy@est.tech>

### CODE REVIEW

**Comment by pperiyasamy** [comment]
I'm still having problem in getting ovs instance up and running with forwarder pod, not enough logs available from artifacts, working with @Bolodya1997 and @denis-tingaikin on possible ways of troubleshooting it further.

**Comment by pperiyasamy** [comment]
@Bolodya1997 @denis-tingaikin I've rebased this to latest ovs deployment-k8s changes and uses github.com/Bolodya1997/cloudtest/tree/debug/no-cluster-cleanup , but now NSE fails to register its service with registry. This causes kernel test to fail though ovs forwarder logs looks so clean during its bringup.  
Any idea what's going on here ?

File: main_test.go [hunk 1]
```diff
@@ -35,6 +38,17 @@ func TestSRIOV(t *testing.T) {
...
 
+func TestOVS(t *testing.T) {
+	ovsSuite := new(ovs.Suite)
+	ovsSuite.SetT(t)
+	// run only Kernel2Kernel test for now
+	ovsSuite.Run("TestKernel2Kernel", func() {
+		ovsSuite.SetupSuite()
+		defer ovsSuite.TearDownSuite()
+		ovsSuite.TestKernel2Kernel()
+	})
```
Comment (denis-tingaikin) [inline main_test.go:49]: For debug you can use a pkg `flag` and testify.m option.

Note: you can use a hack with pkg `flag`

```go
f := flag.Lookup("testify.m")
require.NoError(t, flag.Set("testify.m", "TestKernel2Kernel"))
defer func() { _= flag.Set("testify.m", f.Value)}()
suite.Run(t, new(ovs.Suite))
```

Comment (pperiyasamy) [reply main_test.go]: @denis-tingaikin done, let me check test run now.

**Comment by ljkiraly** [comment]
Follow up in #394

**Pull request closed without merging**